server/komodo-rest/src/main/java/org/komodo/rest/service/KomodoDataserviceService.java

```diff
@@ -217,7 +217,7 @@ public Response getDataservices( final @Context HttpHeaders headers,
                 throw ( KomodoRestException )e;
             }
 
-            return createErrorResponse(mediaTypes, e, DATASERVICE_SERVICE_GET_DATASERVICES_ERROR);
+            return createErrorResponse(Status.FORBIDDEN, mediaTypes, e, DATASERVICE_SERVICE_GET_DATASERVICES_ERROR);
         }
```

Comment:
I see this as a regression that does not need changing. It means that all calls to createErrorResponse() have to include their own Status. Now most will be FORBIDDEN but that's no longer mandated and opens up the possibility that some future REST calls could start returning a different Status. It sounds odd maybe but the previous version mandates a certain prescribed status for the response which is approved by the http status RFCs.